Collision/src/main/java/dk/gruppeseks/bodtrd/collision/CollisionProcessor.java

```diff
@@ -76,17 +77,22 @@ else if (ent.getType() == EntityType.WALL)
             }
             case PROJECTILE:
             {
-                if (ent.getType() == EntityType.PLAYER)
+                if (ent.getState() == EntityState.DESTROYED)
```

Comment:
EntityState.Destroyed should preferably be checked when checking if objects are colliding instead of here